projects/gnomad/src/client/DownloadsPage.js

```diff
@@ -365,13 +234,24 @@ export default () => (
         <h3>Gene constraint</h3>
         <ul>
           <li>
-            <ExternalLink href="https://console.cloud.google.com/storage/browser/gnomad-public/release/2.1/ht/constraint">
-              Hail Table (gs://gnomad-public/release/2.1/ht/constraint/constraint.ht)
+            <ExternalLink href="https://console.cloud.google.com/storage/gnomad-public/release/2.1.1/constraint/gnomad.v2.1.1.lof_metrics.by_transcript.ht">
```

Comment:
For other Hail tables on the page, links point to the table's parent directory instead of the table directory itself.

For example, https://console.cloud.google.com/storage/browser/gnomad-public/release/2.1.1/ht/exomes instead of gs://gnomad-public/release/2.1.1/ht/exomes/gnomad.exomes.r2.1.1.sites.ht